src/components/FolioNumberInput/FolioNumberInput.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <v-form id="folio-number-form" class="ml-8" ref="folioForm" v-model="folioFormValid">
+      <v-text-field
+        filled
+        id="folio-number-textfield"
+        label="Folio Number (Optional)"
+        :value="folioNumber"
+        :rules="folioNumberRules"
+        :disabled="disabled"
+        @input="emitFolioNumber($event)"
+      />
```

Comment:
Should you use prop `persistent-hint` ?

You probably should add `autocomplete="chrome-off"` and `:name="Math.random()"` props to prevent form auto-fill.

Reply:
I added it to prevent auto-fill. Thanks.